src/grip/Grip.story.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+import { action } from '@storybook/addon-actions'
+
+import { ItemRadio } from '../itemRadio'
+import { ItemRadioGroup } from '../itemRadioGroup'
+import { TheVoice } from '../theVoice'
+import { Grip } from './index'
+
+<Meta title="Widgets|Grip" />
+
+# Grip
+
+<Preview>
+  <Story name="Default">
+    <Grip onSlideUp={action('Slide Up')} onSlideDown={action('Slide Down')}>
```

Comment:
Not a biig fan of the naming, If we put actual content in the Grip, IMO it's not a grip anymore but more a container being decorated with a Grip and the Grip naming is misleading.

Reply:
Yeah, it's a bit of the stretch, but it comes from the way I've decided to split up the responsibilities of the components that I'll build for the final screen.
From a technical point of view, I wanted the `Grip` component to carry the responsibility of handling the gestures, and since the specs mention that a larger area than the small `Grip` is responsive to finger gestures (in my case, the whole routes list) I needed the `Grip` to be able to have children to expand the area that is "grippable".
I could have done something like:
```
<GripContainer>
  <Grip />
  {children}
</GripContainer>
```
But I preferred to have the main component that we are going to use in our code `Grip` for consistency with the specs.